src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @include flex(center, '');
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(/src/images/star.svg);
```

Comment:
The path to the image should be relative to the CSS file location. Ensure that the path '/src/images/star.svg' is correct and accessible from the CSS file's location.